onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -479,14 +482,14 @@ def inc_dataloader():
         del dataloader
         model = sq.transform(extra_options.get("SmoothQuantAlpha", 0.5), extra_options.get("SmoothQuantFolding", True))
         sq_path = tempfile.TemporaryDirectory(prefix="ort.quant.")
-        model_input = Path(sq_path.name).joinpath("sq_model.onnx").as_posix()
-        model.save(model_input)
+        sq_model_path = Path(sq_path.name).joinpath("sq_model.onnx").as_posix()
+        model.save(sq_model_path)
         nodes_to_exclude.extend([i.name for i in model.model.graph.node if i.name not in orig_nodes])
-        model = load_model_with_shape_infer(Path(model_input))  # use smooth quant model for calibration
+        model = load_model_with_shape_infer(Path(sq_model_path))  # use smooth quant model for calibration
 
     with tempfile.TemporaryDirectory(prefix="ort.quant.") as quant_tmp_dir:
         calibrator = create_calibrator(
-            Path(model_input),
+            Path(sq_model_path),
```

Comment:
if smoothQuant is false, there will be no a sq_model_path

Reply:
Yeah that's true I missed that. I'll change it back.